.gitignore

```diff
@@ -4,3 +4,6 @@ dist/
 
 /regal
 /regal.exe
+
+input.json
```

Comment:
Might be good to add a comment where these come from single it's not obvious?

Reply:
Yeah, good call, I'll add that!